packages/suite/src/actions/wallet/transactionActions.ts

```diff
@@ -78,7 +77,7 @@ export const remove = (account: Account, txs: WalletAccountTransaction[]): Trans
  * @param {string} newTxid
  */
 export const replaceTransaction =
-    (account: Account, tx: PrecomposedTransactionFinal, newTxid: string) =>
+    (_account: Account, tx: PrecomposedTransactionFinal, newTxid: string) =>
```

Comment:
What about removing the `_account` param altogether?

Reply:
Yes, I shall do it. I just hope that it won't create another conflict with @juriczech's branch?

Reply:
could be followup :+1:

Reply:
You can remove it here if you want as there will be conflicts to my PR anyway and I will fix them when this will be merged to develop. Or I can remove this param when resolving conflicts. Either way is fine with me. @marekrjpolak 

https://tppr.me/ijQqj